test/tests/partitioners/petsc_partitioner/tests

```diff
@@ -0,0 +1,137 @@
+[Tests]
+  [./ptscotch]
+    requirement = 'MOOSE need more partitioning options'
```

Comment:
These are not good requirement statements. I would suggest saying something like this instead.
"MOOSE shall support the <partitioner> mesh partitioner"

Requirements should be specific, testable, unique, etc:
https://qracorp.com/write-clear-requirements-document/